packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -33,6 +33,7 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 	protected readonly blobsShaCache = new Map<string, string>();
 	private readonly summaryTreeUploadManager: ISummaryUploadManager;
 
+	// eslint-disable-next-line @typescript-eslint/class-literal-property-style
 	public get repositoryUrl(): string {
```

Comment:
Does this need to be a getter? I think we can replace this with `public readonly repositoryUrl = "";`

Reply:
We can, but it could be considered an API change. At least api-extractor's output changes. While this is targeting `next` right now, I don't think I'll get it merged before main and next converge, so I don't want to make any public API changes here.

Reply:
Ah, yeah I guess it would be breaking for anyone extending the class. Fair enough.

Reply:
I made these changes separately in #17495.